src/list/snippets/list-item.html

```diff
@@ -0,0 +1,33 @@
+<!-- List with action -->
+<div class="mdl-cell mdl-cell--6-col">
+  <h4> List Item </h4>
+  <ul class="mdl-list">
+    <li class="mdl-list__item">
+      <span class="mdl-list__item-primary-content">
+        <i class="material-icons  mdl-list__item-icon">person</i>
+        Bryan Cranston
+      </span>
+      <span class="mdl-list__item-secondary-action">
+          <input type="checkbox" id="checkbox-1" class="mdl-checkbox__input" checked />
```

Comment:
Any inputs used in the demos should use our full components. That way we know if any modifications are needed for displaying them within the component. As it is, these are just using standard HTML inputs which is not by the specification.